lib/BuildSystem/BuildSystem.cpp

```diff
@@ -47,6 +47,7 @@
 #include <memory>
 #include <mutex>
 #include <set>
+#include <sstream>
```

Comment:
Please don't introduce uses of `sstream`, we use LLVM's `raw_ostream` which can do the same stuff but more cheaply.